graphjet-core/src/test/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountTest.java

```diff
@@ -330,40 +330,87 @@ public void testTopSecondDegreeByCountWithSocialProofUsersWithSmallGraph() throw
     Map<Byte, Integer> minUserPerSocialProof = new HashMap<>();
     List<UserRecommendationInfo> expectedTopResults = new ArrayList<>();
 
+    byte[] socialProofTypes = new byte[] {0, 1, 2, 3};
+    boolean isOnlyUseSpecifiedProofTypes = false;
```

Comment:
1) It might be a good time to split this file. Probably keep this one for generic testing and also add one for Tweet and one for User. I want block this change, but maybe a follow up just to refactor the Test files?

2) Rename to onlyUseSpecifiedProofTypes?